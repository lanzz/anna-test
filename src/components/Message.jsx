import React from 'react';

export default function Message({ts, from, children}) {
    return (
        <div className={`message from-${from}`}>
            <div className="ts">Sent on {String(ts)}</div>
            <div className="content">{children}</div>
        </div>
    );
}
