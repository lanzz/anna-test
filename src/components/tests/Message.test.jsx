import React from 'react';
import { render } from 'enzyme';
import Message from '../Message';

it('renders correctly', () => {
    const wrapper = render(
        <Message />
    );
    expect(wrapper).toMatchSnapshot();
});
