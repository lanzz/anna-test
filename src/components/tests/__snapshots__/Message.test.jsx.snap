// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="message from-undefined"
>
  <div
    class="ts"
  >
    Sent on undefined
  </div>
  <div
    class="content"
  />
</div>
`;
